grype/vulnerability/namespace.go

```diff
@@ -55,6 +55,8 @@ func distroNamespace(d distro.Distro) string {
 		case distro.Alpine:
 			// XXX this assumes that a major and minor versions will always exist in Segments
 			return fmt.Sprintf("alpine:%d.%d", versionSegments[0], versionSegments[1])
+		case distro.Windows:
+			return "microsoft"
```

Comment:
question: should we make this more constrained? something like `microsoft:kb`? In case we run into other microsoft package namespaces

Reply:
Is there such a thing as a "microsoft package" ?

Reply:
https://docs.microsoft.com/en-us/windows/package-manager/winget/

>Windows Package Manager and the winget tool are in public preview and may be substantially modified before they are generally available.

I think it makes sense to narrow down the namespace based on what we know today. If this is strictly MSRC using KBs, it'd be great to communicate this somehow.

Reply:
Unfortunately what this is, is the way we have currently to relate a feed from a Linux Distribution version - it is *not* a way to accurately define a _type_ or something constrained. 

If we ever find ourselves in a situation that a Linux Distribution like Debian has more than one feed associated with it aside from the version, we are going to get into trouble.

If I made the change here to force it to be `"microsoft:kb"` then I would need to make a translation back to `"microsoft"` back again since the feed namespace is `"microsoft"`.  I'm not seeing what the benefit of the constraint is other than semantics.

Reply:
Discussed this offline with @luhring and got to the conclusion that we must infer a namespace from a distro (or OS in this case) that maps to a feed service namespace. I've added clarification to the doc comment and made the return value a constant.